lib/visit_details.dart

```diff
@@ -0,0 +1,165 @@
+              const SizedBox(height: 5.0),
+              Container(
+                decoration: _greyStyledBox,
+                padding: const EdgeInsets.all(15),
+                child: const Text(
+                    "Knee Fracture and had a bad back pain for the last 2 weeks.",
+                    style: TextStyle(
+                      fontWeight: FontWeight.bold,
+                      fontSize: 14,
+                      height: 1.5,
+                    )),
+              ),
+              const SizedBox(height: 15.0),
+              _titleText("Treatment"),
+              const SizedBox(height: 5.0),
+              Container(
+                decoration: _greyStyledBox,
+                padding: const EdgeInsets.all(15),
+                child: const Text(
+                    "Doctors performed knee surgery and gave pain killer.",
+                    style: TextStyle(
+                      fontWeight: FontWeight.bold,
+                      fontSize: 14,
+                      height: 1.5,
+                    )),
+              ),
+              const SizedBox(height: 15.0),
+              //TODO: Ask John about styling here: Keep as is or make it look like others?
+              Container(
+                decoration: _greyStyledBox,
+                padding: const EdgeInsets.all(15),
+                child: const Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    Text("Total Time",
+                        style: TextStyle(
+                          fontStyle: FontStyle.italic,
+                          fontSize: 14,
+                          height: 1.5,
+                        )),
+                    Text(
+                      "2 hrs 45 minutes",
+                      style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                        fontSize: 14,
+                        height: 1.5,
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              const SizedBox(height: 15.0),
+              _titleText("Care Takers"),
+              const SizedBox(height: 5.0),
+              Container(
+                decoration: _greyStyledBox,
+                padding: const EdgeInsets.all(15),
+                child: Column(
+                  children: [
+                    _careTakerRow("Eric", "X-Ray"),
+                    const SizedBox(height: 5.0),
+                    _careTakerRow("Steve", "Lab Attendee"),
+                    const SizedBox(height: 5.0),
+                    _careTakerRow("Shelly", "Triage"),
+                    const SizedBox(height: 5.0),
+                    _careTakerRow("Dr. Sinek", "Doctor"),
+                  ],
+                ),
+              ),
+              const SizedBox(height: 15.0),
+              _titleText("Timeline"),
+              const SizedBox(height: 5.0),
+              Container(
```

Comment:
maybe what we did with the painter should be the column of all the details as its own widget.  Then that can be used both in the visit details page and for this part here.